metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/service/zoho/model/ZohoOrganizationAdapter.java

```diff
@@ -79,14 +79,12 @@ public ZohoOrganizationAdapter(JsonNode response)
 	 * @return The value of Zoho 'content' field for passed 'val' field
 	 */
 	private String getContent(String fieldName) {
```

Comment:
I still do not really agree with this kind of parsing. Doing a getContent(and searching a list of fields) for every time a field is requested seems slower than just converting to a java object at the beginning and mapping all the required fields to the class fields. I understand it's not a huge impact since this happens in memory for not too many fields and probably only once for each field.
That's my opinion though and I reported this in the previous review, so if you still insist keeping it as we can proceed in merging.

Reply:
I would avoid the double mapping. JsonNode ->ZohoOrganization->EdmOrganization, especially becasue the first maping cannot be done automatically due to the weird/generic structure of Zoho output. We wanted to implement like that by using automic mapping, but it doesn't work for this case. I still think here make sense to put the code maintenance efforts above performance consideration, as we don't have any performance issues on such tiny objects.

Reply:
here is how the zoho response looks like
"FL":[{"val":"ACCOUNTID","content":"1482250000002112001"},{"val":"SMOWNERID","content":"1482250000002063029"},{"val":"Account Owner","content":"Annapaula Freire"},{"val":"Account Name","content":"National Library of France"},{"val":"Website","content":"www.bnf.fr"}